src/helpers/Functions.js

```diff
@@ -28,3 +28,13 @@ export const filterDateTime = (timestamp) => {
   }
   return dateTime;
 };
+
+export const capilizeFirstLetter = (name) =>
+  name.charAt(0).toUpperCase() + name.slice(1);
+
+export const userFullName = (hasUser, firstName, middleName, lastName) =>
```

Comment:
`userFullName` should not care if user exists, it should only use the last 3 params and format the user's name. Do the validation outside of the function